examples/rgbpp_transfer.py

```diff
@@ -0,0 +1,96 @@
+from io import BytesIO
+from buidl import HDPrivateKey, NamedHDPublicKey, PSBT, read_varstr, TxFetcher
+import requests
+import threading
+import time
+
+from rgbpp.types import RgbppTransferReq, Hex
+from rgbpp.rpc import rpc
+
+INTERVAL_TIME_SECONDS = 30
+# Please use your own BTC seed to generate private key
+SEED = b'Hello RGB++'
+
+# Please make sure you have enough BTC and RGB++ Assets, the example is only for RGB++ transfer on BTC.
+# You can get RGB++ assets using the [rgbpp-sdk examples](https://github.com/ckb-cell/rgbpp-sdk/tree/develop/examples/rgbpp/xudt)
+def transfer_rgbpp_on_btc(params: RgbppTransferReq):
+    # Please make sure the newtork of BTC is correct, including mainnet, testnet and signet
+    network = "signet"
+
+    # HD Key (note the named_key path only contains 4 layers)
+    root_key = HDPrivateKey.from_seed(SEED, network)
+    root_named_key = NamedHDPublicKey.from_hd_priv(root_key, "m/84'/0'/0'")
+
+    # P2WPKH Address
+    p2wpkh_path = "m/84'/0'/0'/0"
+    p2wpkh_key = root_key.traverse(p2wpkh_path)
+    p2wpkh_child_0 = p2wpkh_key.child(0)
+    print(f'BTC P2WPKH Address: {p2wpkh_child_0.p2wpkh_address()}')
+
+    # Generate RGB++ CKB virtual tx and BTC PSBT hex from the rgbpp-sdk-service
+    response = rpc.generate_rgbpp_transfer_tx(params)
+    btc_psbt_hex = response['btc_psbt_hex']
+    ckb_virtual_tx_result = response['ckb_virtual_tx_result']
+    print(f'CKB virtual tx result: {ckb_virtual_tx_result}')
+
+    # Parse PSBT
+    psbt = PSBT.parse(BytesIO(bytes.fromhex(btc_psbt_hex)), network)
+    
+    # Update lookup context
+    tx_lookup = psbt.tx_obj.get_input_tx_lookup()
+    pubkey_lookup = root_named_key.bip44_lookup()
+    psbt.update(tx_lookup, pubkey_lookup)
+
+    # Sign PSBT with the root_key
+    psbt.sign(root_key)
+
+    # Finalize PSBT and convert to TX
+    psbt.finalize()
+    btc_tx = psbt.final_tx()
+    signed_tx_hex = btc_tx.serialize().hex()
+    print(f"BTC signed tx: {signed_tx_hex}")
+
+    # Broadcast the BTC signed transaction
+    btc_tx_id = TxFetcher.sendrawtransaction(signed_tx_hex, network)
+    print(f"BTC tx id: {btc_tx_id}")
```

Comment:
It's not using assets-api to broadcast the BTC transaction?

BTC assets-api will broadcast a BTC transaction to multi BTC nodes/electrs.

Reply:
Broadcasting BTC transactions with the assets-api is better.

Reply:
I'm not sure and the `TxFetcher.sendrawtransaction` is provided by `buidl-python` to broadcast BTC tx.
```python 
    "mainnet": "https://blockstream.info/api",
    "testnet": "https://blockstream.info/testnet/api",
    "signet": "https://mempool.space/signet/api",
}
```
We can also use BTC Assets API(wrapped by SDK Service) to broadcast BTC tx.

Reply:
OK. I will use BTC Assets API to broadcast BTC tx later